tests/apollo/util/bft.py

```diff
@@ -1834,9 +1834,30 @@ def reset_metadata(self, replica_id):
                             str(self.config.c),
                             "37",  # number of principles, defined in setup.cpp
                             "10"]  # max client batch size, defined in ReplicaConfig.hpp
+            out_file_name = "reset_md_out." + str(replica_id) + ".log"
+            err_file_name = "reset_md_err." + str(replica_id) + ".log"
+            with open(out_file_name, 'w+') as stdout_file, open(err_file_name, 'w+') as stderr_file:
+                res = subprocess.run(reset_md_cmd, stdout=stdout_file, stderr=stderr_file)
+                assert res.returncode == 0
+
+    def remove_metadata(self, replica_id):
+        with log.start_action(action_type="remove_metadata", replica_id=replica_id) as action:
+            db_editor_path = os.path.join(self.builddir, "kvbc", "tools", "db_editor", "kv_blockchain_db_editor")
+            db_dir = os.path.join(self.testdir, DB_FILE_PREFIX + str(replica_id))
+            remove_md_cmd = [db_editor_path,
+                            db_dir,
+                            "removeMetadata",
+                            str(replica_id),
+                            str(self.config.f),
+                            str(self.config.c),
+                            "37",  # number of principles, defined in setup.cpp
```

Comment:
Do we have a way importing and not mentioning explciit values? This is less maintainable...the 10 and 37.

Reply:
Thanks. Removed these 2 as they were not necessary.